### PR TITLE
Updates to torch/padding

### PR DESCRIPTION
**Pull request by MMathisLab**
Hi @Hvitgar -- not sure these ever made it back into the bethge repo; but I had issues back in the day with padding of images and needing updates to torch.nn; not sure it's useful anymore (but this is the branch I need to use to make it work). 

-mackenzie

### CODE REVIEW

**Comment by bmitzkus** [comment]
these changes are already part of master =)

**Comment by MMathisLab** [comment]
awe great; yeah I thought it was odd what files it showed would be edited in the PR, as I thought I has changed this a long time ago :). thanks!

**Comment by bmitzkus** [comment]
This is because it was not rebased to the current state of master and therefore compared to a version of this repo from may 2019. As master is identical to your changes, I assume that at some point in 2019 we already merged your changes =)
